debian/postinst

```diff
@@ -20,6 +20,7 @@ db_get geotrek-admin/TIME_ZONE; export TIME_ZONE="$RET"
 db_get geotrek-admin/LANGUAGES; export LANGUAGES="$RET"
 db_get geotrek-admin/SERVER_NAME; export SERVER_NAME="$RET"
 db_get geotrek-admin/RANDO_SERVER_NAME; export RANDO_SERVER_NAME="$RET"
+export RANDO_SERVER_NAME_PIPE="`echo "$RANDO_SERVER_NAME" | sed 's/ /|/g'`"
```

Comment:
An HTTP request with `Origin: http://foo?http://valid-rando-domain` will be allowed by the regexp in the map if `valid-rando-domain` is set in $RANDO_SERVER_NAME. The regexp in $RANDO_SERVER_NAME_PIPE should strictly match Rando URLs, so I suggest you enclose the regexp with `^$` only if not empty and not `*`:

```suggestion
if [ -n "${RANDO_SERVER_NAME}" ] && [ "${RANDO_SERVER_NAME}" != "*" ]; then
    export RANDO_SERVER_NAME_PIPE="^`echo "$RANDO_SERVER_NAME" | sed 's/ /|/g'`$"
else
    export RANDO_SERVER_NAME_PIPE="`echo "$RANDO_SERVER_NAME" | sed 's/ /|/g'`"
fi
```